scripts/bootstrap

```diff
@@ -8,4 +8,4 @@ fi
 python3 ./manage.py makemigrations
 python3 ./manage.py migrate
 python3 ./banwebScrape.py
-python3 ./manage.py custom_db_sync ./writer.csv
+python3 ./manage.py custom_db_sync ./transfer-info.csv
```

Comment:
Probably should add `transfer-info.csv` and `transfer-info.json` to `.gitignore`.